src/queue.ts

```diff
@@ -19,8 +19,8 @@ export class HelpQueueDisplayManager {
         const status_line = `The queue is **${queue.is_open ? 'OPEN' : 'CLOSED'}**. There ${quant_prase} in the queue.\n`
         if(queue.length > 0) {
             const table = new AsciiTable()
-            table.setHeading('Position', 'Name')
-            queue_members.forEach((state, idx) => table.addRow(idx + 1, state.member.user.username))
+            table.setHeading('Position', 'Username', 'Nickname')
+            queue_members.forEach((state, idx) => table.addRow(idx + 1, state.member.user.tag, state.member.user.username))
```

Comment:
I think you're looking for [displayName](https://discord.js.org/#/docs/discord.js/stable/class/GuildMember?scrollTo=displayName). I think it's redundant to have a username and nickname. Maybe just use displayName.

If you want to include this in a separate PR that's ok too. Technically it should be but I won't complain if you make that change here.

Reply:
I think i did something wrong, since I hadn't meant for this to be in this pull request. I'll remove it